.ci/install_kata_image.sh

```diff
@@ -167,8 +211,14 @@ get_dependencies() {
 
 main() {
 	get_dependencies
-	local os_version=$(get_version "${IMAGE_OS_VERSION_KEY}")
-	local osbuilder_distro=$(get_version "${IMAGE_OS_KEY}")
+	if [ "${BUILD_WITH_DRACUT}" == "yes" ]; then
+		local os_version="${VERSION_ID}"
+		local osbuilder_distro="${ID}"
+		local build_method_suffix=".dracut"
```

Comment:
The CI is faling because it's caught a bug in your script :smile: This line creates a variable that is only defined if `${BUILD_WITH_DRACUT}" == "yes"`... but the script then uses this variable later, even though it may not have been set.

Reply:
Ah right, thanks for pointing that out!  I missed the `nounset` at the top of the script I guess and then I missed the "unbound variable" message in the log... a double whammy :-( .